src/client/automation/playback/press/key-press-simulator.js

```diff
@@ -142,7 +152,8 @@ export default class KeyPressSimulator {
         var raiseDefault  = eventSimulator.keyup(getDeepActiveElement(this.topSameDomainDocument), extend(eventOptions, modifiersState));
         var activeElement = getDeepActiveElement(this.topSameDomainDocument);
 
-        if (!browserUtils.isFirefox && raiseDefault && this.sanitizedKey === 'space' &&
+        if (!browserUtils.isFirefox && !browserUtils.isSafari && (!browserUtils.isWebKit || browserUtils.version >= 53) &&
```

Comment:
@AlexanderMoskovkin, i think there is no need for the test, cause original reason for changing this rule was [this test](https://github.com/DevExpress/testcafe/blob/master/test/client/fixtures/automation/key-press-simulation-test.js#L316) - it was failed